include/trace/events/sof.h

```diff
@@ -0,0 +1,38 @@
+#undef TRACE_SYSTEM
```

Comment:
@nklayman You need to follow the guidelines for the commit title. This need to be titled
'ASoC: SOF: add initial tracing support

This patch introduces the use of trace points in the SOF driver, with the ability to trace widget setup and free.
'